swift/modules/SwiftUtilities/Tests/SwiftUtilitiesTests/TestWaiterTests.swift

```diff
@@ -0,0 +1,48 @@
+/*
+   Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+   SPDX-License-Identifier: Apache-2.0
+*/
+
+import XCTest
+@testable import SwiftUtilities
+
+final class TestWaiterTests: XCTestCase {
+    /// Create a waiter with default settings and use it; make sure task result
+    /// is as expected
```

Comment:
/// Create a waiter with default settings and use it. Make sure that the
    /// task result is as expected.